docs/guides/data-work/update-data.md

```diff
@@ -205,8 +186,10 @@ After your updates, the old steps are no longer relevant. Therefore, we move the
 
 You have now completed the first iteration of your work. Time to get a second opinion on your changes!
 
-!!! note "The PR to review is merging the _review_ branch into the _reference_ branch"
-    Your current draft PR (called "📊 Update Near-surface temperature anomaly data") attempts to merge the sub-branch `update-temperature-anomaly` into `temp-update-temperature-anomaly`.
+!!! note "Helping reviewer see the actual changes"
+    Github code diff will include copied files as new files, and the reviewer will not be able to distinguish what is copied and what is new.
+
+    Command `etl draft-pr ... --step-update` has added a link _View diff without step copy_ to your PR description. It shows code changes excluding the first commit that copied the steps. If it stops working, it's likely because you rebased your branch. In that case, replace commit hash with the one that copied the steps.
```

Comment:
I think it's common that we will rebase at some point while a PR is open. Does it screw up the review?
If so, maybe explain a bit better what you mean by "replace commit hash".

Reply:
If you rebase, the automatically created link showing diffs without the first commit will stop working (because rebasing changes commit hashes). In that case, just update the commit hash in PR description.

I've explained this more clearly.